src/Query.tsx

```diff
@@ -37,8 +36,9 @@ export interface FetchMoreQueryOptions<TVariables, K extends keyof TVariables> {
 // Improved ObservableQuery field types, need to port them back to Apollo Client
 export type ObservableQueryFields<TData, TVariables> = Pick<
   ObservableQuery<TData>,
-  'startPolling' | 'stopPolling'
+  'startPolling' | 'stopPolling' | 'subscribeToMore'
```

Comment:
SubscribeToMore is used for subscriptions right? Should it be exposed in the Query component or is it being used for backwards compatibility?

Reply:
If we do decide to keep it then I think we should include a test in the `Query.test.tsx` file in a follow-up

Reply:
@excitement-engineer for sure! It isn't really tested in the current test suite either, just a smoke test to make sure its found and callable